libs/ui/select/helm/src/lib/hlm-select-trigger.component.ts

```diff
@@ -1,18 +1,41 @@
-import { Component, ContentChild, type ElementRef, ViewChild, computed, input } from '@angular/core';
+import { Component, ContentChild, type ElementRef, ViewChild, computed, input, } from '@angular/core';
```

Comment:
Small detail not sure why the comma here at the end with "input". Mainly surprised biome didn't pick it up or complain about it haha